.github/workflows/nightly_tests_and_veracode.yml

```
@@ -68,6 +67,12 @@ jobs:
           summary_display: true
           baseline_file: baseline.json
           include: "alfresco-hxinsight-connector*"
+      - name: "Notify on Veracode SAST scan failure"
+        if: failure()
+        uses: Alfresco/alfresco-build-tools/.github/actions/send-teams-notification@v8.14.1
+        with:
+          webhook-url: ${{ secrets.MS_TEAMS_WEBHOOK_URL }}
+          message: "Nightly Veracode SAST scan failed"

```

I think that this also runs on push and pull requests to master.

---

here we expect that notification will be sent for "if" conditions. Same notification action is used for nightly_tests

---

notification condition is updated to run for schedule event in:
https://github.com/Alfresco/hxinsight-connector/pull/889/commits/755aa8416e89b5c4bc68591c06c7421492f79e08

---

teams notification make sense only for SCA while it runs in scheduled workflow

for SAST as it's related to application code should report any failure during PRs (may need to handle a base list for known issues initially)

---

Veracode SAST job was moved to ci.yml in order to run on every PR. 
Work on SCA job will be done in separate PR as it is getting an error:
https://github.com/Alfresco/hxinsight-connector/actions/runs/13800635533